content/docs/services/elasticsearch24.md

```diff
@@ -67,7 +67,7 @@ You can rotate credentials by creating a new instance and deleting the existing
     cf create-service-key my-elasticsearch-backup my-key
     ```
 
-* Connect to your Elasticsearch service using port forwarding. Note: You'll need to leave the `cf ssh` command running and follow the next steps in a different terminal so that you can access the remote Elasticsearch instance from your local environment:
+* Connect to your Elasticsearch service using port forwarding. Note: You'll need to leave the [`cf ssh`]({{< relref "docs/apps/using-ssh.md" >}}) command running and follow the next steps in a different terminal so that you can access the remote Elasticsearch instance from your local environment:
```

Comment:
Do we want to add this for the Redis docs as well? It's the same process for Redis as well regarding terminal lifetime and port-forwarding.

Reply:
Sure! That sounds like it could be based on the [gist linked here](https://gsa-tts.slack.com/archives/C09CR1Q9Z/p1505768537000429?thread_ts=1505757511.000129&cid=C09CR1Q9Z) - I'm not exactly sure how to integrate it into the Redis page though.

Reply:
I'll take a crack at it then and update the PR. someone else should approve it then.

Reply:
(Maybe good for that to be a separate PR? Sounds like that would be its own mini-project.)

Reply:
:smile: oops just saw this. I just borrowed from what was already there. I didn’t want to recommend any particular tools around how to back it up. But I figured the SSH portforwarding would be useful since it came up during my support rotation.